### DISCUSSION TITLE
Implementing a search box for TMDB

### DISCUSSION THREAD

**Discussion by kaaloo** [observablehq/framework#871] [Q&A]
Hi,

I would like to implement a simple search box that queries TMDB for recent films.  It seems that it's not possible to pass a parameter to a data loader right?  What mechanism would you suggest?

**Comment by Fil** [comment]
Note that you don't _have to_ use data loaders, which capture data snapshots at build time. In the case you're describing, I would probably use `fetch` (or `d3.json`) on the API endpoint, and pass it the values the user entered in an input — that request would come from the client (user browser), it would not be made when the site is built.

**Reply by kaaloo** [reply]
Thanks that makes perfect sense!
